src/objectTranslations/Data_Import_Object_Mapping__mdt-ja.objectTranslation

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- Data Import Object Mapping --></value>
+    </caseValues>
+    <fields>
+        <help><!-- If custom apex logic should be used to map the fields for this object, then this field should be populated with the name of a class (including namespace if needed) that extends the BDI_ObjectMappingLogic class and overrides the populateObject method. --></help>
+        <label><!-- Custom Mapping Logic Class --></label>
+        <name>Custom_Mapping_Logic_Class__c</name>
+    </fields>
+    <fields>
+        <help><!-- The Data Import Object Group containing this record. --></help>
+        <label><!-- Data Import Object Mapping Set --></label>
+        <name>Data_Import_Object_Mapping_Set__c</name>
+        <relationshipLabel><!-- Data Import Object Mappings --></relationshipLabel>
+    </fields>
+    <fields>
+        <help><!-- The Field API Name on the NPSP Data Import object where the ID for the imported record will be placed when the import is successful. This is also where the code will verify if this record was imported or matched. --></help>
+        <label><!-- Imported Record Field Name --></label>
+        <name>Imported_Record_Field_Name__c</name>
+    </fields>
+    <fields>
+        <help><!-- The Field API Name where the success or failure of the import will be recorded for an NPSP Data Import record. --></help>
+        <label><!-- Imported Record Status Field Name --></label>
+        <name>Imported_Record_Status_Field_Name__c</name>
+    </fields>
+    <fields>
+        <label><!-- Is Deleted --></label>
```

Comment:
@ptewson-sfdo - Dropping these suggestions in case we do want help text for this field.

```suggestion
        <help><!-- When checked, indicates this record was deleted from the Data Import Advanced Mapping user interface and will be ignored during NPSP Data Import processing. --></help>
        <label><!-- Is Deleted --></label>
```